internal/assertions/assertions.go

```diff
@@ -162,6 +162,23 @@ func ErrorIs(err error, target error) (s string) {
 	return
 }
 
+func ErrorAs[E error, Target *E](err error, target Target) (s string) {
+	if err == nil {
+		s = "expected error; got nil\n"
+		return
+	}
+	if target == nil {
+		s = "expected target not to be nil"
```

Comment:
Maybe this, wording is a pain

```suggestion
		s = "expected not-nil error as target"
```

But at least it would Match the behavior of `Error`function

Reply:
All the Error assertions were worded differently, so I changed them all to make them consistent.